src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
+<script context="module">
+  export async function preload() {
+    const posts = await this.fetch(`blog.json`)
+      .then((r) => r.json())
+      .then((posts) => posts);
+    const events = await this.fetch(`events.json`)
+      .then((r) => r.json())
+      .then((events) => events);
+    const data = await this.fetch("index_temp.json")
+      .then((r) => r.json())
+      .then((data) => data);
+    return {
+      posts,
+      events,
+      cardsData: data.cardsData,
+      revealNewHomepage: data.revealNewHomepage,
+    };
+  }
+</script>
+
 <script>
-  import { Grid, Row, Column } from "carbon-components-svelte";
-  import { Banner, SidebarRight } from "~/partials";
+  import { Button } from "carbon-components-svelte";
+  import { ArrowRight16 } from "carbon-icons-svelte";
   import { Card, CardsContainer } from "~/components/cards";
+  import { Banner, SidebarRight } from "~/partials";
+  import { InlineNotification } from "carbon-components-svelte";
+  import InformationFilled32 from "carbon-icons-svelte/lib/InformationFilled32";
 
-  const dummyLink = "/homepage-redesign-preview/#";
-
-  const cardsData = [
-    {
-      titleText: "Tools, How to Guide, & Data Download",
-      linkPath: "/",
-      description: `Access climate data via visualizations, downloads, and
-        the Cal-Adapt API for your research, outreach, and adaptation planning
-        needs.`,
-      bgColor:
-        "linear-gradient(180deg, #0F9699 0%, #0F9982 0.01%, #035466 100%)",
-      ctaText: "Learn More",
-    },
-    {
-      titleText: "Cal-Adapt: Analytics Engine",
-      linkPath: "https://analytics.cal-adapt.org",
-      description: `Utilize cloud storage and computation to analyze Petabytes
-        of stored data supported by scientific guidance.`,
-      bgColor: "linear-gradient(180deg, #248DB3 0%, #096180 100%)",
-      ctaText: "Visit Site",
-    },
-    {
-      titleText: "Data Development Grants",
-      linkPath: dummyLink,
-      description: "Description forthcoming.",
-      bgColor: "linear-gradient(180deg, #19953E 0%, #196B5B 100%)",
-      ctaText: "Learn More",
-    },
-  ];
-  const cardHeight = 20;
-  const cardWidth = 18;
+  export let events;
+  export let posts;
+  export let cardsData;
+  export let revealNewHomepage;
```

Comment:
Add a new variable e.g. `cardsDataTemp` to store the updated language for the 3 cards.